docs: UIPlugin updates (#546)
rhobs/observability-operator: docs/user-guides/observability-ui-plugins.md

  type: TroubleshootingPanel
```

#### Feature Matrix

| __COO Version__ |   __OCP Versions__  | __Features__                                          |
| --------------- | ------------------- | ----------------------------------------------------- |
| 0.3.0+          | 4.16+               | _No features configuration, just core functionality_  |

### Distributed Tracing

> The plugin adds tracing related UI features to the OpenShift web console. A new tab can be located in the admin perspective under `Observe > Traces`. This tab allows a user to select a TempoStack running in their cluster as well as a set a time range and query for the traces being loaded. These traces are displayed on a scatter-plot showing the trace start time, duration, and number of spans. Underneath the scatter plot there is a list of traces showing information such as the `Trace Name`, number of `Spans`, and `Duration`. The trace name contains a link which takes the user to the trace detail page for the selected trace containing a Gantt Chart timeline of all of the spans within the trace. Once selected, the spans show a breakdown of their configured attributes.

Reviewer: ```suggestion
The plugin adds tracing related UI features to the OpenShift web console. A new tab can be located in the admin perspective under `Observe > Traces`. This tab allows a user to select a Tempo instance running in their cluster as well as a set a time range and query for the traces being loaded. These traces are displayed on a scatter-plot showing the trace start time, duration, and number of spans. Underneath the scatter plot there is a list of traces showing information such as the `Trace Name`, number of `Spans`, and `Duration`. The trace name contains a link which takes the user to the trace detail page for the selected trace containing a Gantt Chart of all of the spans within the trace. Once selected, the spans show a breakdown of their configured attributes.
```

Author: Changed in the latest commit

spec:
  type: DistributedTracing
```

#### Feature Matrix

| __COO Version__ |   __OCP Versions__  | __Features__                                          |
| --------------- | ------------------- | ----------------------------------------------------- |
| 0.3.0+          | 4.11+               | _No features configuration, just core functionality_  |

### Logging

> The plugin adds various Logging UI functionalities to the OpenShift web console. The core functionality of this plugin adds a new admin perspective tab `Observe > Logging`. This page includes query and log filters, the list of logs, and a histogram showing log frequency by severity. The logs can be filtered by an number of tags, such as `tenant` and `namespace`. The page also has controls for the length of time to query over, the refresh rate of the logging page, and whether to show kubernetes resource information in the results, such as `pod` and `container`. The reults section shows a list of collapsed logs, which can then be expanded to show more detailed information for each log.

Reviewer 2: ```suggestion
The plugin adds various Logging UI functionalities to the OpenShift web console. The core functionality of this plugin adds a new admin perspective tab `Observe > Logs`. This page includes query and log filters, the list of logs, and a histogram showing log frequency by severity. The logs can be filtered by an number of tags, such as `tenant` and `namespace`. The page also has controls for the length of time to query over, the refresh rate of the logging page, and whether to show kubernetes resource information in the results, such as `pod` and `container`. The results section shows a list of collapsed logs, which can then be expanded to show more detailed information for each log.
```

Author: fixed in most recent commit

> When a __TroubleshootingPanel__ `UIPlugin` is deployed the plugin will connect the [Korrel8r](https://github.com/korrel8r/korrel8r) service and add direct links from the admin perspective `Observe > Logging` page to the `Observe > Metrics` page with a correlated PromQL query. It will also add a "See Related Logs" link from the admin perspective `Observe > Alerting` on an alerting detail page to the `Observe > Logging` page with a correlated filter set selected.

Reviewer 2: ```suggestion
When a __TroubleshootingPanel__ `UIPlugin` is deployed the plugin will connect the [Korrel8r](https://github.com/korrel8r/korrel8r) service and add direct links from the admin perspective `Observe > Logs` page to the `Observe > Metrics` page with a correlated PromQL query. It will also add a "See Related Logs" link from the admin perspective `Observe > Alerting` on an alerting detail page to the `Observe > Logs` page with a correlated filter set selected.
```
